capstone/src/main/java/com/google/step/servlets/PageViewServlet.java

```diff
@@ -15,16 +15,16 @@
 import com.google.step.data.RestaurantPageViews;
 
 /**
- * Servlet that will call the metricsClient for any doGet or doPost requests.
+ * Servlet that will read and write page view metrics to and from a persistent store.
  */
 @WebServlet("/page-view")
 public class PageViewServlet extends HttpServlet {
 
-    private static MetricsClient metricsClient = new MetricsClient();
+    private final MetricsClient metricsClient = new MetricsClient();
     private static Gson gson = new Gson();
```

Comment:
hehe this too, any reason to keep it static? :P

Reply:
I promise I have eyes haha. Fixed.